<?xml version="1.0" ?>

<launch>    
    <param name="gazebo"                   value="true"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_op3"/>
    
    <param name="offset_file_path"         value="$(find op3_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find op3_manager)/config/OP3.robot"/>
    <param name="init_file_path"           value="$(find op3_manager)/config/dxl_init_OP3.yaml"/>
    <param name="device_name"              value="/dev/ttyUSB0"/>
    
    <!-- OP3 Manager -->
    <node name="op3_manager" pkg="op3_manager" type="op3_manager" output="screen"/>

    <!-- OP3 Optimization -->
    <node name="op3_optimization" pkg="op3_optimization" type="op3_optimization" output="screen"/>
    
</launch>
